package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headquarters-node",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
because, in a way, this is a breaking change, because it's using a different endpoint by default.
If we for some reason had both endpoints working separately, with different data, we wouldn't want to force clients to move to the new one with just a patch update

Of course, this is not the case, since the API is the same. But that's a problem of the API itself, not the wrapper.

Also, some things might break, such as the email validations, which are now `subvisual.co` instead of `groupbuddies.com`